src/components/ContentExplorer/ContentExplorer.js

```diff
@@ -1325,6 +1344,14 @@ class ContentExplorer extends Component<Props, State> {
                             onItemPreview={this.preview}
                             onSortChange={this.sort}
                         />
+                        <Footer>
+                            <Pagination
```

Comment:
Will we always render pagination in the explorer? In webapp we don't show anything if the entries fit on one page.

Reply:
The pagination controls aren't rendered if there's only one page and the footer is hidden via the `:empty` pseudo-class in that case.